packages/core/scss/global/elements/_elements.lists.scss

```
@@ -62,3 +62,12 @@ ol > li {
     font-size: 16px;
   }
 }
+blockquote > ol {
+  position: relative;
+  left: 50px;
+}
+@include mq($until: desktop) {
+  blockquote > ol {
+    left: -20px;
+  }
+}
```

Litt usikker, men dette kan kanskje forkortes slik?

```suggestion
blockquote > ol {
  position: relative;
  left: 50px;
  @include mq($until: desktop) {
    left: -20px;
  }
}
```